### ISSUE TITLE
Time out appraisals

### ISSUE THREAD

**Issue by Claudweb** [evepraisal/go-evepraisal#46]
You added the option to create non-persistant appraisals and also to delete appraisals. Most of the appraisals I am doing are only interesting for a limited time but during that time they need to be persisted. I also don't want to delete each of them manually because this would be to much effort, so they are kept and will clutter my history and your db. 

How about specifying a default timeout per user? After that configured time the appraisel then could be safely removed from the system.

**Comment by sudorandom** [comment]
Yeah, this wouldn't be too hard to add. Thanks for the thought and suggestion.

**Comment by sudorandom** [comment]
Also note that appraisals will eventually be purged after 90 days of no activity (no one viewing the appraisal).
